### DISCUSSION TITLE
Can pika stop running task?

### DISCUSSION THREAD

**Discussion by weidongzhou1994** [pika/pika#1424] [Q&A]
As the title, celery can stop the running task, can pika do it?

**Comment by lukebakken** [comment]
> As the title, celery can stop the running task, can pika do it?

Celery and Pika are two different things, so you'll have to explain a bit more. Celery is a work scheduler that uses RabbitMQ, and Pika is an AMQP library.

I suppose you could write an application that uses Pika to publish a message that a Celery worker could understand, but I don't know how Celery works so I can't answer that.

You may wish to ask here - https://github.com/celery/celery/discussions

PS don't use GitHub issues for questions when the Discussions feature is available.
